engine/platform/irix/dladdr.c

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Can you add copyright/GPL notice and short description? You can copypaste it from other files. Like this: https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/zone.c#L1